index.html

```diff
@@ -7,12 +7,29 @@
 </head>
 <body>
   <header>
-    <h2>Tech Lancaster</h2>
+    <h2 class="header1">Tech Lancaster</h2>
   </header>
-  <section>
+  <section class="s1">
     <img src="" alt="">
-    <h1>Suppprting the tech community in lancaster.
+    <h1 class="header2">Suppprting the tech community in lancaster.
     </h1>
   </section>
+  <section class="s2">
+    <p class="layer-1">TechLancaster started as a simple local meetup, a safe place for like-minded tech people to come together and geek out.</p>
+
+    <p class="layer-2">We've strived to be open, welcoming, and supportive to everyone in the community, regardless of their interest or background.
+
```

Comment:
looks good just one questions are you trying to make a break line in this empty space?